eq-author-api/utils/questionnaireEvents.js

```diff
@@ -10,23 +17,25 @@ const questionnaireCreationEvent = (questionnaire, ctx) => ({
 
 const noteCreationEvent = (ctx, bodyText) => ({
   id: uuid.v4(),
-  publishStatus: ctx.questionnaire.publishStatus,
+  publishStatus: translate[ctx.questionnaire.publishStatus],
   questionnaireTitle: `${ctx.questionnaire.title} (Version ${ctx.questionnaire.surveyVersion})`,
   bodyText,
   userId: ctx.user.id,
   time: new Date(),
 });
 
-const changedPublishStatusEvent = (ctx, questionnaireVersion) => ({
+const publishStatusEvent = (ctx, bodyText) => ({
   id: uuid.v4(),
-  publishStatus: ctx.questionnaire.publishStatus,
-  questionnaireTitle: `${ctx.questionnaire.title} (Version ${questionnaireVersion})`,
+  publishStatus: translate[ctx.questionnaire.publishStatus],
```

Comment:
`publishStatus` in other places are expecting the non translated version, and translation is done on frontend. For example on the table of questionnaires. Should we keep it that way?

Reply:
I think we've already chatted about that and I agreed the translation should happen on the front end. Otherwise I'm coupling the backend to the frontend and that seems wrong. I'll make the translation happen on the frontend.